nuntium/tests/answers_test.py

```diff
@@ -7,10 +7,10 @@
 class AnswerTestCase(TestCase):
     def setUp(self):
         super(AnswerTestCase, self).setUp()
-        self.message = Message.objects.all()[0]
-        self.person = Person.objects.all()[0]
+        self.message = Message.objects.get(id=1)
+        self.person = Person.objects.get(id=1)
         #There is no membership for this guy to any writeitInstance
-        self.person_not_in_the_instance = Person.objects.all()[1]
+        self.person_not_in_the_instance = Person.objects.get(id=2)
```

Comment:
And this one? I'm just slightly concerned because I was assuming you were getting them back in primary key order in your environment.